tests/unit/checkDeployBlockersTest.ts

```diff
@@ -35,10 +42,12 @@ beforeAll(() => {
             },
         },
     };
+
+    // @ts-expect-error TODO: Remove this once GithubUtils (https://github.com/Expensify/App/issues/25382) is migrated to TypeScript.
```

Comment:
Could you please add a comment in that issue noting that this will need to be updated once it's been migrated?

Reply:
There is always a clean up issue at the end with the intention of going through all this TODOs and remove them.